app/models/list.rb

```diff
@@ -1,7 +1,11 @@
 class List < ActiveRecord::Base
-  has_many :list_entries,class_name:'ListEntry'
+  has_many :list_entries, class_name: 'ListEntry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.